flagscale/serve/run_inference_engine.py

```diff
@@ -45,7 +69,7 @@ def main():
 
     model_config = None
     for item in serve_config:
-        if item.get("serve_id", None) == "vllm_model":
+        if item.get("serve_id", None) in ["vllm_model", "llama_cpp_model"]:
```

Comment:
I believe the serve_id distinguishes the current serving from others. So, why do we only allow vllm_model or llama_cpp_model?